src/test/c_api_tests.cpp

```diff
@@ -41,14 +44,54 @@ static void testDefaultSingleModelOptions(MultiModelOptionsImpl* mmo) {
     EXPECT_EQ(mmo->idleSequenceCleanup, std::nullopt);
 }
 
+#define ASSERT_CAPI_STATUS_NULL(code)         \
```

Comment:
Rename code -> error or ovmsError. Code could be mistaken for StatusCode

Reply:
This in fact is a code which get called inside the macro to get the code. Should I really rename it to ovmsError?